### PR TITLE
1.4.0.Final announcement

### PR DESCRIPTION
**Pull request by Naros**

### CODE REVIEW

File: _posts/2021-01-07-debezium-1-4-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+author: ccranfor
+---
+
+I am pleased to announce the release of Debezium *1.4.0.Final*!
+
+This release concludes the major work put into Debezium over the last three months.
+Overall, the community fixed https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20in%20(1.4.0.Final%2C%201.4.0.Alpha1%2C%201.4.0.Beta1%2C%201.4.0.CR1)%20ORDER%20BY%20issuetype%20DESC%2C%20updated%20DESC%2C%20priority%20DESC[117 issues] during that time, including the following key features and changes:
+
+* New link:/documentation/reference/connectors/vitess.html[Vitess] connector
+* Fine-grained Selection of Snapshotted Tables
```
Comment (gunnarmorling) [inline _posts/2021-01-07-debezium-1-4-final-released.adoc:15]:
```suggestion
* Fine-grained selection of snapshotted tables
```

File: _posts/2021-01-07-debezium-1-4-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: ccranfor
+---
+
+I am pleased to announce the release of Debezium *1.4.0.Final*!
+
+This release concludes the major work put into Debezium over the last three months.
+Overall, the community fixed https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20in%20(1.4.0.Final%2C%201.4.0.Alpha1%2C%201.4.0.Beta1%2C%201.4.0.CR1)%20ORDER%20BY%20issuetype%20DESC%2C%20updated%20DESC%2C%20priority%20DESC[117 issues] during that time, including the following key features and changes:
+
+* New link:/documentation/reference/connectors/vitess.html[Vitess] connector
```
Comment (gunnarmorling) [inline _posts/2021-01-07-debezium-1-4-final-released.adoc:14]: Also worth linking the dedicated blog post by Kewei

File: _posts/2021-01-07-debezium-1-4-final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* Fine-grained Selection of Snapshotted Tables
+* PostgreSQL `Snapshotter` completion hook
+* Distributed link:/blog/2020/12/16/distributed-tracing-with-debezium/[Tracing]
+* MySQL support for _create_ or _read_ records emitted during snapshot
+* Many Oracle link:/documentation/reference/connectors/oracle.html#_logminer[Logminer adapter] improvements
```
Comment (gunnarmorling) [inline _posts/2021-01-07-debezium-1-4-final-released.adoc:19]: LM support was new to begin with in 1.4, right?

File: _posts/2021-01-07-debezium-1-4-final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* Full support for Oracle JDBC connection strings
+* Improved reporting of DDL errors
+
++++<!-- more -->+++
+
+Please refer to previous release announcements (https://debezium.io/blog/2020/10/23/debezium-1-4-alpha1-released/[Alpha1], https://debezium.io/blog/2020/11/17/debezium-1-4-alpha2-released/[Alpha2], https://debezium.io/blog/2020/12/09/debezium-1-4-beta1-released/[Beta1], https://debezium.io/blog/2020/12/17/debezium-1-4-cr1-released/[CR1]) for more details.
```
Comment (gunnarmorling) [inline _posts/2021-01-07-debezium-1-4-final-released.adoc:25]: URLs should start with link:

File: _posts/2021-01-07-debezium-1-4-final-released.adoc [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+https://github.com/rareddy[Ramesh Reddy],
+https://github.com/rgannu[Ganesh Ramasubramanian],
+https://github.com/seeekr[Denis Andrejew],
+https://github.com/telnicky[Travis Elnicky],
+https://github.com/vanhoale[Hoa Le],
+https://github.com/yimingl17[Yiming Liu],
+https://github.com/yrodiere[Yoann Rodière], and
+https://github.com/zrlurb[Peter Urbanetz].
+
+Overall, more than 220 individuals have contributed to the Debezium project and the number of Debezium link:/community/users/[users] continues to grow.
```
Comment (gunnarmorling) [inline _posts/2021-01-07-debezium-1-4-final-released.adoc:61]: I got 245 names in the contributors file, so let's use that number? You also could link to yesterday's post btw.

**Review by gunnarmorling — COMMENTED** [review]
LGTM, thanks! A few comments inline. Free free to merge once those are addressed. Thx!

File: _posts/2021-01-07-debezium-1-4-final-released.adoc (hunk 3 above)
Comment (Naros) [reply _posts/2021-01-07-debezium-1-4-final-released.adoc:19]: No, we added it late in 1.3 but continued to stablize & improve it during 1.4.

**Pull request merged**